task_sdk/src/airflow/sdk/execution_time/comms.py

```diff
@@ -70,6 +71,7 @@ class StartupDetails(BaseModel):
 
     Responses will come back on stdin
     """
+    ti_context: TIRunContext | None = None
```

Comment:
Can this ever actually be None?

Reply:
Nope, only in tests. Will change it to reflect the reality -- was convenient for tests

Reply:
Updated